guide/03-the-gis/cloning-content.ipynb

```diff
@@ -0,0 +1,2051 @@
+{
```

Comment:
<p>Just curious, can an administrator clone items belonging to another administrator? Also, is there any requirement for source administrator and target administrator to be having the same user name, and other license types/privileges/roles?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<ul><li>I tested - logged in as one administrator, I could clone the items of another administrator.</li><li>There is no requirement for the usernames of the administrators to match. The source code gets username values form the gis object.</li><li>The source code checks to ensure the target user has these privileges depending upon the item type: <em>portal:user:createItem, portal:user:createGroup, or portal:publisher:publishFeatures.  </em>The clone_items() function will fail if the necessary privilege is not present when cloning the respective item type.</li></ul>

Reply:
<p>A user can clone any items they have access to. So if items are shared publicly i can clone them from any org to my org. We are just using the sharing model, so if it is shared with me I can clone it, but as John described I can't create anything that I am don't have the privileges to do. So if I can create hosted feature layers through the UI, I can't clone them.</p>

Reply:
<p>Added a note in the guide to explain this:</p><p> </p><p><strong><em>NOTE:</em></strong><em>&nbsp;Cloning relies on the sharing model to determine the items any particular&nbsp;</em><code><a href="https://develoipers.arcgis.com/python/api-reference/arcgis.gis.toc.html#user" target="_blank"><em>user</em></a></code><em>&nbsp;can clone. If a user can access an item, that user can clone it. However, that user can't create anything in the target portal they don't have the appropriate&nbsp;</em><code><a href="https://enterprise.arcgis.com/en/portal/latest/administer/windows/roles.htm#ESRI_SECTION1_4FF9051EFB814C249AB26B0ACFF7C79F" target="_blank"><em>privileges</em></a></code><em>&nbsp;to create.</em></p>